pubspec.yaml

```diff
@@ -4,10 +4,10 @@ version: 4.0.0
 homepage: https://github.com/luanpotter/ordered_set
 
 environment:
-  sdk: '>=2.12.0 <3.0.0'
+  sdk: '>=2.14.0 <3.0.0'
 
 dev_dependencies:
-  test: ^1.16.0-nullsafety
-  dart_code_metrics: ^3.2.2
-  dartdoc: ^0.42.0
+  test: ^1.17.10
+  dart_code_metrics: ^4.6.0
```

Comment:
replace with flame_lint maybe? (can be followup of course)

Reply:
I was thinking that too, just didn't want to put too much in this PR. :)